editor/src/clj/editor/workspace.clj

```diff
@@ -361,16 +369,11 @@ ordinary paths."
       (g/update-property workspace :resource-types editable-resource-type-map-update-fn resource-types-by-ext)
       (g/update-property workspace :resource-types-non-editable non-editable-resource-type-map-update-fn resource-types-by-ext))))
 
-(defn- editability->output-label [editability]
-  (case editability
-    :editable :resource-types
-    :non-editable :resource-types-non-editable))
-
```

Comment:
Logic moved to the `resource/resource-types-by-type-ext` function, which now accepts `:editable`, `:non-editable`, or booleans for the `editability` argument.